### DISCUSSION TITLE
iOS Client

### DISCUSSION THREAD

**Discussion by trymeouteh** [ImranR98/Obtainium#1758] [Ideas]
Add support for iOS for downloading sideloaded version of iOS apps from sources like Github, Gitea, Gitlab, Forgejo, etc.

Glad the app is written in Flutter to make such as thing easier, not having to build another app from scratch

**Comment by ImranR98** [comment]
https://github.com/ImranR98/Obtainium/discussions/1760#discussioncomment-10195371
